guide/designing-products/user-research.md

```diff
@@ -84,21 +83,24 @@ There are many ways in which research and development are similar to other digit
    layout = "float-right-desktop"
 %}
 
-Most digital products start with a specific country, culture, language or legal framework in mind. This is especially true for financial applications, which are typically highly regulated and need to slowly and strategically evolve. While Bitcoin did start with a single language, its use case was not tied to any particular context, and this has resulted in a much more diverse user base from the start.
+When designers begin to develop a product, they often start with a specific country, culture, language, or legal framework in mind. This is especially true for financial applications, which are highly regulated and need to evolve slowly and strategically. While Bitcoin was originally developed for an English speaking audience, it didn’t take long for its intended use case to naturally proliferate it.
```

Comment:
Are you sure you meant to use `proliferate` here?

Reply:
Reproduce rapidly, spread, so yeah it makes sense to me, or do you have another better-suited word in mind?

Reply:
It just changes the original point. This new version states that Bitcoin got quickly adopted by english-speaking audiences because it was first released in English. The original point was that Bitcoin is not limited by context (local laws, culture, borders...), which allowed it to have a diverse user base right away (meaning people all over the world could use it) - despite the original limitation of being English-only. That builds on the argument of this paragraph that researchers need to think differently about their users - global and diverse from the start vs start local and become global. Does that make sense?

Reply:
Agree with @GBKS, doesn't read as clearly or tie in as nicely with the rest of the paragraph.

Reply:
@GBKS @Bosch-0 Reverted in https://github.com/BitcoinDesign/Guide/pull/365/commits/66c0c6f3821190af411958971a2342857cfa0f3a